packages/core-mobile/scripts/bitrise/detox/iosInternalE2e.sh

```diff
@@ -3,6 +3,10 @@
 # make pipelines' return status equal the last command to exit with a non-zero status, or zero if all commands exit successfully
 set -o pipefail
 
+echo "Here is the public IP address of the machine running the tests"
+
+curl -s http://ipecho.net/plain; echo
```

Comment:
is it intended to do in iOSInternalE2e.sh?

Reply:
i don't know why this change keeps coming up.  i'll remove it.